tests/test_user/test_login.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
```

Comment:
This file should be `tests/users/test_login_views`

Format is `tests/<app_name>/test_<what_you_are_testing>`

Reply:
Changes done